ape/ape_prompts/gen-instructions.prompt

```diff
@@ -47,7 +47,7 @@ If you would like to use few shot examples, use the reserved variable {{_FEWSHOT
 Use the information below to learn about a task that we are trying to solve using calls to an LM, then write a new prompt that will be used to prompt a Language Model to better solve the task.
 
 Task for LLM:
-{prompt_desc}
```

Comment:
this is meant to be `prompt_desc` to describe the task of the prompt. task_description that comes below as a suggestion is to provide a tip (especially for human expertise)

Reply:
right now, there is no way of adding prompt_desc into optimize function, and that is why i removed it. 
i'll add prompt_desc as a param to mipro.optimize and revive the variable